test/fixtures/template/template.jade

```diff
@@ -0,0 +1 @@
+p= item.title
```

Comment:
Yup, good catch. That's a remnant from the switch from regular jade to posthtml. Will remove.